maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/StatelessXmlReporter.java

```diff
@@ -240,6 +240,10 @@ public void testSetCompleted( WrappedReportEntry testSetReportEntry, TestSetStat
             }
             ppw.endElement(); // TestSuite
         }
+        catch ( final IOException e )
+        {
+            throw new ReporterException( "Failure generating XML report.", e );
```

Comment:
I started a surefire build and then left because it runs a long time and I did not want to wait. There were a couple of ITs no longer failing. I'll write an email when getting home. Results looked promising on OpenBSD. Last commit is just temporary for testing. I closed various maven-shared-utils issues today which seem to fix things for surefire as well. I'll send an email when I am back. I hope the console will show a successful build. 

[CommandLineUtils](http://svn.apache.org/viewvc?view=revision&revision=1784432)
[StreamFeeder](http://svn.apache.org/viewvc?view=revision&revision=1784431)